matrix/system_of_linear_equation_in_2_variables.py

```diff
@@ -0,0 +1,84 @@
+def calculate_x_and_y(eq1, eq2):
```

Comment:
```suggestion
def calculate_x_and_y(eq1: list[int], eq2: list[int]) -> str:
```

Reply:
what should I type in place of str if it will raise only runtime error (according to @CaedenPH 's changes) ??

Reply:
> what should I type in place of str if it will raise only runtime error (according to @CaedenPH 's changes) ??

Runtime error doesnt count as a return type, it implies a return type of `None`

Reply:
Exceptions that are raised are NOT return values.  The way to think about this is that if the function succeeds (no Exceptions are raised) then the call should expect to receive a str.